beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -36,6 +36,11 @@ pub const ENGINE_GET_PAYLOAD_TIMEOUT: Duration = Duration::from_secs(2);
 pub const ENGINE_FORKCHOICE_UPDATED_V1: &str = "engine_forkchoiceUpdatedV1";
 pub const ENGINE_FORKCHOICE_UPDATED_TIMEOUT: Duration = Duration::from_millis(500);
 
+pub const ENGINE_EXCHANGE_TRANSITION_CONFIGURATION_V1: &str =
+    "engine_exchangeTransitionConfigurationV1";
+pub const ENGINE_EXCHANGE_TRANSITION_CONFIGURATION_V1_TIMEOUT: Duration =
+    Duration::from_millis(100);
```

Comment:
This seems like a really short timeout, particularly if the EL and CL are on different machines/continents. I'd be tempted to say all timeouts should be at least 600ms given that 1 roundtrip round the earth is ~500ms.

Reply:
Good point, I hadn't considered this very well. I've changed it to 500ms to give a little room to move :)